src/utils/media-utils.js

```diff
@@ -0,0 +1,21 @@
+import { MEDIA_TYPE } from '../domain/constants.js';
+
+const audioUrlExtensions = ['aac', 'm4a', 'mp3', 'oga', 'ogg', 'wav', 'flac'];
+const videoUrlExtensions = ['mp4', 'm4v', 'ogv', 'webm', 'mpg', 'mpeg', 'mov', 'avi', 'mkv'];
+
+export const getMediaType = url => {
+  const sanitizedUrl = (url || '').trim();
+  const extensionMatches = sanitizedUrl.match(/\.([0-9a-z]+)$/i);
+  const extension = extensionMatches && extensionMatches[1];
```

Comment:
Just to be on the safe side I'd propose `const extension = extensionMatches?.[1]?.toLowerCase();` realizing that some applications produce files like `MYSOUND.MP3`

Reply:
Oh of course, the regex that was in the markdown plugin was case insensitive 🤦‍♀️ 
Fixed, thanks for noticing it!